src/components/LowerRightControls.tsx

```diff
@@ -56,7 +56,7 @@ export function LowerRightControls({
           Element="externalLink"
           to={getReleaseUrl()}
           className={
-            '!no-underline !border-none font-mono text-xs' +
+            '!no-underline !border-none !bg-transparent font-mono text-xs' +
```

Comment:
This is the actual fix, mind the background color in dark mode behind the version

<img width="1650" alt="image" src="https://github.com/user-attachments/assets/1982a62b-9844-4148-b5df-5c6e0d55e938" />